src/utils/ProfileUtils.ts

```diff
@@ -200,8 +203,12 @@ export async function populateImportedProfile(
 
```

Comment:
Nitpick: I lowkey don't like it how all `progressCallbacks` used to have the same function signature but now others take a number while this one still takes a string. Consistency would be nice.

Reply:
Should be more consistent again. An extra parameter still had to be added to `installModsToProfile`'s `progressCallback`.